### PR TITLE
6 publish on firefox

### LINKED ISSUE #6: Publish on firefox
- [x] Setup the file structure
- [ ] Create an account on Mozilla so we can publish
- [ ] Publish the extension

### PR DESCRIPTION
**Pull request by MSaurus**
A first version that can be published on Firefox

Resolves #6

### CODE REVIEW

**Comment by MSaurus** [comment]
Need to update README.md with instructions on how to use Mozillas `web-ext` tool to start the extension and to build the extension.

File: js/todo.js
Comment (jonathan-lindqvist) [inline js/todo.js:56]: Something is funky when using the default theme, when creating todos it will change the color to white. Like this;
bug

**Review by jonathan-lindqvist — CHANGES_REQUESTED** [review]
Looks good overall, just some small bugs/improvements 👍

**Review by jonathan-lindqvist — APPROVED** [review]
Looks good!

File: README.md
Comment (jonathan-lindqvist) [inline README.md:33]: Understand what you are trying to convey but maybe it word it differently? Maybe "Latest node.js version (14.x.x.)"

**Review by jonathan-lindqvist — CHANGES_REQUESTED** [review]
Look over the only comment, once "fixed" you can merge without another review!

**Pull request merged**
